.drone.yml

```
@@ -2,25 +2,31 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-name: license
+name: Check License Header Presence
 kind: pipeline
 type: docker
 
+clone:
+  depth: 1
+
 steps:
-  - name: check
+  - name: Check License Header Presence
     image: docker.io/library/golang:1.20
     pull: always
     commands:
       - go install github.com/google/addlicense@v1.1.1
-      - addlicense -c "SIGHUP s.r.l" -v -l bsd -y "2017-present" --check .
+      - addlicense -c "SIGHUP s.r.l" -v -l bsd --check .
```

Don't remove the year flag:
```suggestion
      - addlicense -c "SIGHUP s.r.l" -v -l bsd -y "2017-present" --check .
```